src/Sulu/Bundle/AdminBundle/Resources/public/js/components/login/main.js

```diff
@@ -352,382 +352,421 @@ define([], function() {
+            this.sandbox.dom.on(this.dom.$forgotPasswordFrame, 'keydown', this.forgotPasswordKeyHandler.bind(this));
+            this.sandbox.dom.on(this.dom.$requestResetMailButton, 'click', this.requestResetMailButtonClickHandler.bind(this));
+
+            // reset error-status on user input-element-change
+            this.sandbox.dom.on(this.dom.$forgotPasswordFrame, 'keyup change', this.validationInputChangeHandler.bind(this, this.dom.$forgotPasswordFrame), '.husky-validate');
         },
 
         /**
-         * Handles the window's mousenter-event
-         * @param event
+         * Bind resend-mail-frame related dom events
          */
-        mouseenterHandler: function(event) {
-            if (event.relatedTarget === null) {
-                this.mouseOrigin.x = event.pageX;
-                this.mouseOrigin.y = event.pageY;
-            }
+        bindResendMailDomEvents: function () {
+            this.sandbox.dom.on(this.dom.$resendResetMailButton, 'click', this.resendResetMailButtonClickHandler.bind(this));
         },
 
         /**
-         * Handles the window's mousemove-event
-         * @param event
+         * Bind reset-password-frame related dom events (reset mode)
          */
-        mousemoveHandler: function(event) {
-            var changeInX = (event.pageX - this.mouseOrigin.x) * -this.movementRatio.x,
-                changeInY = (event.pageY - this.mouseOrigin.y) * -this.movementRatio.y;
-            this.setBackgroundPosition(changeInX, changeInY);
-            this.mouseenterHandler(event);
+        bindResetPasswordDomEvents: function () {
+            this.sandbox.dom.on(this.dom.$resetPasswordButton, 'click', this.resetPasswordButtonClickHandler.bind(this));
+            this.sandbox.dom.on(this.dom.$resetPasswordFrame, 'keydown', this.resetPasswordKeyHandler.bind(this));
+            this.sandbox.dom.on(this.sandbox.dom.find('.' + constants.loginRouteClass), 'click', this.loginRouteClickHandler.bind(this));
+
+            // reset error-status on user input-element-change
+            this.sandbox.dom.on(this.dom.$resetPasswordFrame, 'keyup change', this.validationInputChangeHandler.bind(this, this.dom.$resetPasswordFrame), '.husky-validate');
         },
 
         /**
-         * Handles a click on the login-button
+         * Handle click on login-button in login-frame
          */
-        loginButtonClickHandler: function() {
+        loginButtonClickHandler: function () {
             this.sandbox.dom.submit(this.dom.$loginForm);
         },
 
         /**
-         * Handles the click on the login-route item (reset-mode)
+         * Handle change of validation-input-element value on given frame
+         * @param $frame parent frame of changed input-element
          */
-        loginRouteClickHandler: function() {
-            this.redirect(this.options.loginUrl);
+        validationInputChangeHandler: function ($frame) {
+            if (event.keyCode === 13) {
```

Comment:
@nnatter doesn't this cause an error like 'cant get property keyCode of undefined'. Maybe $frame should be 'event'?